akka-docs/build.sbt

```diff
@@ -34,6 +34,10 @@ paradoxProperties ++= Map(
   "extref.github.base_url" -> (GitHub.url(version.value) + "/%s"), // for links to our sources
   "extref.samples.base_url" -> "https://github.com/akka/akka-samples/tree/2.5/%s",
   "extref.ecs.base_url" -> "https://example.lightbend.com/v1/download/%s",
+  "scaladoc.akka.base_url" -> "https://doc.akka.io/api/akka/current",
+  "scaladoc.akka.http.base_url" -> "https://doc.akka.io/api/akka-http/current",
+  "javadoc.akka.base_url" -> "https://doc.akka.io/japi/akka/current",
+  "javadoc.akka.http.base_url" -> "http://doc.akka.io/japi/akka-http/current",
```

Comment:
Wow we didn't have that yet? crazy :).

While pointing to `current` could lead to dead links in outdated documentation, it still seems like the right thing to do :+1: .

Reply:
might be right for http, but not sure about `scaladoc.akka.base_url` and `javadoc.akka.base_url`. Wouldn't it be better to use 2.5 there so that old 2.5 docs are self contained when we release 2.6 as current?

Reply:
Hmm, yes perhaps people will google anyway instead of bookmarking these days. We should make sure google indexes 'current', but we can solve that with 'canonical' headers/tags.

Reply:
I agree that linking to 2.5 so we're self contained should be safer here; Shall I change that and merge @raboof @patriknw ?